src/main.go

```diff
@@ -242,7 +242,10 @@ func cleanup() {
 		if node.state == "active" {
 			switch node.service {
 			case "Ping":
-				HandlerPingGeneral("stop")
+				ok := HandlerPingGeneral("stop")
+				if !ok {
+					panic("Error occured while closing modulwe")
+				}
```

Comment:
`ok` is used only once. Hence we can make this to:

```
if ok := HandlerPingGeneral("stop"); !ok {
	panic("Error occured while closing module")
}
```

Reply:
Will do this.